packages/orbit-components/src/Stepper/StepperStateless/index.tsx

```diff
@@ -1,118 +1,33 @@
   if (typeof maxWidth === "string") return maxWidth;
-  return `${parseInt(maxWidth, 10)}px`;
+  return `${parseInt(maxWidth.toString(), 10)}px`;
 };
 
-const StyledStepper = styled.div`
-  display: flex;
-  width: 100%;
-  max-width: ${getMaxWidth};
-  flex: 1 1 auto;
-`;
-
-const iconMixin = css<{ isActive?: boolean; isDisabled?: boolean }>`
-  ${({ theme, isActive, isDisabled }) => css`
-    padding: 2px;
-    height: 20px;
-    width: 20px;
-    background: ${isActive ? theme.orbit.paletteBlueNormal : theme.orbit.paletteCloudNormal};
-    border-radius: ${theme.orbit.borderRadiusCircle};
-    ${mq.desktop(css`
-      height: 16px;
-      width: 16px;
-    `)};
-
-    ${!isDisabled &&
-    css`
-      &:hover {
-        background: ${isActive
-          ? theme.orbit.paletteBlueNormalHover
-          : theme.orbit.paletteCloudNormalHover};
-      }
-
-      &:focus,
-      &:active {
-        box-shadow: inset 0 0 0 2px
-          ${isActive ? theme.orbit.paletteBlueLightActive : theme.orbit.paletteCloudNormalActive};
-      }
-    `};
-  `}
-`;
-
-const StyledMinusIcon = styled(Minus)`
-  ${iconMixin};
-`;
-
-StyledMinusIcon.defaultProps = {
-  theme: defaultTheme,
-};
-
-const StyledPlusIcon = styled(Plus)`
-  ${iconMixin};
-`;
-
-StyledPlusIcon.defaultProps = {
-  theme: defaultTheme,
-};
-
-const StyledStepperInput = styled.input`
-  ${({ theme, disabled }) => css`
-    width: 100%;
-    height: 44px;
-    padding: 0;
-    border: 0;
-    font-size: ${theme.orbit.fontSizeTextLarge};
-    font-weight: ${theme.orbit.fontWeightMedium};
-    color: ${theme.orbit.paletteInkDark};
-    text-align: center;
-    min-width: 0;
-    opacity: ${disabled ? "0.5" : "1"};
-    background-color: ${disabled && "transparent"};
-
-    &::-webkit-inner-spin-button,
-    &::-webkit-outer-spin-button {
-      -webkit-appearance: none;
-      margin: 0;
-    }
-
-    &:focus {
-      outline: none;
-    }
-  `}
-`;
-
-StyledStepperInput.defaultProps = {
-  theme: defaultTheme,
-};
-
-const StyledStepperButton = styled(Button)`
-  &:focus {
-    outline: none;
-
-    ${StyledMinusIcon},
-    ${StyledPlusIcon} {
-      ${defaultFocus}
-    }
-  }
-`;
-
-StyledStepperButton.defaultProps = {
-  theme: defaultTheme,
-};
+const stepperButtonMixin = ({ disabled, active }: { disabled: boolean; active?: boolean }) =>
+  cx(
+    "[&_svg]:p-xxxs",
+    "de:[&_svg]:h-icon-small de:[&_svg]:w-icon-small [&_svg]:h-icon-medium [&_svg]:w-icon-medium",
+    "[&_svg]:rounded-circle",
+    "focus:outline-0",
+    "[&_svg]:focus:outline-blue-normal [&_svg]:focus:outline [&_svg]:focus:outline-2",
+    !disabled && [
+      active
+        ? "[&_svg]:bg-blue-normal hover:[&_svg]:bg-blue-normal-hover [&_svg]:active:focus:shadow-[inset_0_0_0_2px_var(--palette-blue-light-active)]"
+        : "[&_svg]:bg-cloud-normal hover:[&_svg]:bg-cloud-normal-hover [&_svg]:active:focus:shadow-[inset_0_0_0_2px_var(--palette-cloud-normal-active)]",
+    ],
```

Comment:
I think that with these classes it has to be both `active` **and** `focus` at the same time to apply the style, as opposed to the current master where the style is applied when is either `active` **or** `focus`.

Plus, the pseudo-classes here target the button instead of the svg (icon), and then correctly apply the style to the svg.

I believe there's not much difference in terms of behavior when interacting with the component, but it differs from the original components.

Reply:
Said that, I believe we can keep it as it is if changing it would complicate things a lot. In the end, I believe it will be the same for the final user.

Reply:
Thanks! I'd say it's ok to have it like that, if there's no difference